sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosDatabaseRequestOptions.java

```diff
@@ -51,10 +53,16 @@ CosmosDatabaseRequestOptions setOfferThroughput(Integer offerThroughput) {
         return this;
     }
 
+    CosmosDatabaseRequestOptions setThroughputProperties(ThroughputProperties throughputProperties) {
```

Comment:
Between choice of ReqeustOptions vs overload: Isn't overload better from experience wise?

Reply:
Yeah, we use an overload. This is not a public method.